src/App.tsx

```diff
@@ -138,7 +141,18 @@ enum APIRequestStatus {
   'SUCCEEDED',
 }
 
-export default function App() {
+export default function AppWithProvider() {
+  return (
+    <Provider store={store}>
+      <App />
+    </Provider>
+  );
+}
+
+function App() {
+  // manager hooks
+  useAppVersion();
```

Comment:
At the root of `App` we add a reference to a new manager hook which is used for persistent app version polling